.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,31 @@
+name: .NET Core
+
+on: [push]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Chectout
+      uses: actions/checkout@v1
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.0.100
```

Comment:
Instead of hard coding this in the yaml file where it only affects the build system, we should put this in a [global.json file](https://docs.microsoft.com/en-us/dotnet/core/tools/global-json). I'm guessing there's a GH Action to pull the dotnet-version from global.json.

Reply:
I can't find a way right now. Maybe we improve it later.

Reply:
Strange. Azure Pipelines supports this today: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops

Reply:
The Github Actions doesn't support it. https://github.com/actions/setup-dotnet#usage